src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.After;
+import org.junit.Test;
+
+public class StorageTest {
```

Comment:
there is no need to test storage so let's remove this test
if you lack code coverage - see checklist and add this class to excluded in pom.xml